crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -591,12 +591,12 @@ fn dependency_excludes_non_contiguous_range_of_compatible_versions() {
               package-c==1.0.0
               package-c==2.0.0
           and package-c==1.0.0 depends on package-a<2.0.0, we can conclude that package-c<2.0.0 depends on package-a<2.0.0.
-          And because package-c==2.0.0 depends on package-a>=3.0.0, we can conclude that all versions of package-c depend on one of:
+          And because package-c==2.0.0 depends on package-a>=3.0.0, we can conclude that package-c depends on one of:
```

Comment:
Could we consider this change separately? It doesn't seem related to this pull request.